appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -185,12 +210,19 @@ public void deleteUser(@RequestHeader(value = Constants.PASSWORD_HEADER) Optiona
 		return createResponse(user, includePrivateData, HttpStatus.OK);
 	}
 
+	public static Link getUserLinkWithTempPassword(UUID userID, String tempPassword)
```

Comment:
Rename into ``getUserSelfLinkWithTempPassword`` or allow specifying the ``rel`` as parameter.